communications/meetings.md

```diff
@@ -0,0 +1,67 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+
+- Determine the scope and minimum length of the meeting.
+- Give all participants as much context as possible to make an informed decision on RSVP.
+- Gather topics and issues for agenda items that are relevant to the scope of the meeting,
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
```

Comment:
Shall we add that how to use zoom google doc here for the reference?
https://docs.google.com/document/d/1WNqeFKXYv_ji_PUXBKq8nlD09ErqmB6VoWasPW-BUDI/edit?usp=sharing

Reply:
yea, i can link it, but common is public facing.

so we either want to make this google docs public, or we carry the content into common

Reply:
I can make it public. just need @hyx131 's approval since it has her face in it lol

Reply:
alright she approved